stablehlo/tests/stablehlo_legalize_to_vhlo.0_11_0.mlir

```diff
@@ -0,0 +1,2261 @@
+// RUN: stablehlo-opt --mlir-print-op-generic %s.bc | FileCheck %s
```

Comment:
Could you add a negative test like the one here to ensure that this FP type cannot be serialized for v0.10.0:
https://github.com/openxla/stablehlo/blob/main/stablehlo/tests/vhlo_to_version_downgrade_invalid.0_9_0.mlir